saleor/plugins/tests/sample_plugins.py

```diff
@@ -221,6 +221,54 @@ def get_checkout_shipping_tax_rate(
     def get_order_shipping_tax_rate(self, order: "Order", previous_value: Decimal):
         return Decimal("0.080").quantize(Decimal(".01"))
 
+    def get_taxes_for_checkout(
+        self, checkout: "Checkout", previous_value
+    ) -> Optional["TaxData"]:
+        return TaxData(
+            currency=checkout.currency,
```

Comment:
Can we use more realistic values? Like total == subtotal+shipping, subtotal== sum(lines)? We should also use different values for net and gross. Like net == 10.00, gross 12.30

Reply:
It is a detail, but we should have a realistic data in tests.